Makefile

```diff
@@ -111,8 +111,16 @@ KUSTOMIZE_VERSION ?= v5.0.1
 OPM_VERSION ?= v1.27.0
 GO_BINDATA_VERSION ?= v3.1.2+incompatible
 BATS_VERSION ?= 1.2.1
-OLM_VERSION ?= v0.18.2
-KUBERNETES_VERSION ?= v1.26.4
+OLM_VERSION ?= latest
```

Comment:
v0.18.2 gave me OLM CI test error. After I change to latest, then work

Reply:
Looks like there are quite a few releases since `v0.18`, so maybe set it to `v0.25.0`?
https://github.com/operator-framework/operator-lifecycle-manager/releases